easybuild/easyconfigs/g/GDAL/GDAL-2.4.4-foss-2021b.eb

```
@@ -49,7 +49,7 @@ dependencies = [
     ('HDF', '4.2.15'),
 ]
 
-preconfigopts = "sed -e 's/-llapack/\$LIBLAPACK/g' -i.eb configure && "
+preconfigopts = "sed -e 's/-llapack/$LIBLAPACK/g' -i.eb configure && "
```

consistency across `GDAL` easyconfigs
```suggestion
preconfigopts = r"sed -e 's/-llapack/\$LIBLAPACK/g' -i.eb configure && "
```